esphome/components/oxt_dimmer/oxt_dimmer.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+  Copyright © 2023
+*/
+
+/*********************************************************************************************\
+ * The OXT dimmer uses simple protocol over serial @9600bps where each frame looks like:
+ *  0  1  2  3  4  5  6  7  8  9  A  B
+ * 00 00 ff 55                             - Header
+ *             01                          - Channel being updated
+ *                4b                       - Light brightness of channel 1 [00..ff]
+ *                   4c                    - Light brightness of channel 2 [00..ff]
+ *                      05 dc 0a 00 00     - Footer
+\*********************************************************************************************/
+#include "oxt_dimmer.h"
+
+namespace esphome {
+namespace oxt_dimmer {
+
+static const char *const TAG = "oxt";
+
+void OxtController::send_to_mcu_(const OxtDimmerChannel *updated_channel) {
+  struct {
+    uint8_t header[3];
+    uint8_t update;
+    uint8_t channel[2];
+    uint8_t footer[4];
+  } frame{{0x00, 0xff, 0x55}, 0x00, {0x00, 0x00}, {0x05, 0xdc, 0x0a, 0x00}};
+
+  for (size_t index = 0; index < MAX_CHANNELS; index++) {
+    auto *channel = channels_[index];
```

Comment:
> All uses of class members and member functions should be prefixed with this-> to distinguish them from global functions/variables.

https://developers.esphome.io/contributing/code/#c-code-style
```suggestion
    auto *channel = this->channels_[index];
```